src/cow-react/modules/limitOrders/utils/toFraction.ts

```diff
@@ -0,0 +1,16 @@
+import { Fraction } from '@uniswap/sdk-core'
+
+export function toFraction(value: string, isInversed = false) {
+  const [quotient, remainder = ''] = value.split('.')
+
+  if (!value || !Number(value)) return new Fraction(0)
+
+  const numerator = `${quotient}${remainder}`
```

Comment:
I'm sure we don't need to implement this ourselves and theres libraries we can rely that will do a more solid implementation than this. 

Looks very fragile. Also, what if there's thousands separators here?